src/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import Home from './components/Homepage/Home';
+import Homepage from './components/Homepage/Homepage';
+import Detail from './components/Details';
+import Login from './components/LoginRegister/Login';
+import Register from './components/LoginRegister/Register';
+import AddDoctor from './components/AddDoctor/AddDoctor';
+import Auth from './components/LoginRegister/Auth';
+import Appointment from './components/Appointments/Appointments';
+import UserAppointments from './components/Appointments/UserAppointments';
+import Removedoctor from './components/AddDoctor/DeleteDoctor';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.:


![image](https://user-images.githubusercontent.com/57421823/234659788-b6313a7d-61fc-4860-bb01-ebc143ee4044.png)

That way, your code will meet the latest standards. 💪